src/foam/mlang/mlang.js

```diff
@@ -3502,8 +3501,14 @@ foam.CLASS({
       name: 'userContext'
```

Comment:
Why do we need userContext? Why not just use object's get/setX()? Ask Troy.

Reply:
I tried just using the `getX()` originally but it wouldn't work for some reason. I never ended up tracking down exactly why.